### PR TITLE
[LSI-1315] Add export-class support for metadata export

### PR DESCRIPTION
**Pull request by mianchenia**
https://oat-sa.atlassian.net/browse/LSI-1315

Problem: a few times per champing we have transfer all test metadata to customer. For now we have to manually export about 300+ tests and recreate tree structure.

Solution: this PR adds support for export metadata by class. It does minimal changes to get live easier with the simplest solution.

Extra: 
- this changes only for backward support for 2021 release version (DEPP). We got other ticket to make it in better way: https://oat-sa.atlassian.net/browse/REL-880

### CODE REVIEW

**Review by bugalot — APPROVED** [review]
It does the job @mianchenia, good job! Now waiting for a second approval :)

File: models/classes/export/metadata/TestMetadataByClassExportHandler.php
Comment (alroniks) [inline models/classes/export/metadata/TestMetadataByClassExportHandler.php:110]: As export is possible for any class, in the case when nested classes exist two or more tests with the same label, in the archive will be set only the last file. That might produce issues in runtime and increase the possibility of support requests. That's no good.

Comment (mianchenia) [reply models/classes/export/metadata/TestMetadataByClassExportHandler.php]: Thanks! It makes sence. 
I've updated solution. Now it works fully recursive. This case won't be an issue anymore. Please have a look.

**Review by alroniks — APPROVED** [review]
Looks good!

**Pull request merged**
